packages/mint-components/src/components/sqm-link-button/sqm-link-button.feature

```diff
@@ -0,0 +1,15 @@
+@author:noah
+@owner:noah
+
+Feature: Link Button
+
```

Comment:
@noahwc @lillyszatmary where is this even being used?

Reply:
There is often one as a CTA in the friend widget. We should probably add this to that template actually I think?